packages/frontend/amp/components/Body.tsx

```diff
@@ -35,28 +36,39 @@ export const Body: React.FC<{
     pillar: Pillar;
     data: ArticleModel;
     config: ConfigType;
-}> = ({ pillar, data, config }) => (
-    <InnerContainer className={body(pillar)}>
-        <TopMeta config={config} articleData={data} />
-        <Elements
-            pillar={pillar}
-            elements={data.elements}
-            // stuff for ads
-            edition={data.editionId}
-            section={data.sectionName}
-            contentType={data.contentType}
-            switches={config.switches}
-            commercialProperties={data.commercialProperties}
-            isImmersive={data.isImmersive}
-        />
-        <SubMeta
-            sections={data.subMetaSectionLinks}
-            keywords={data.subMetaKeywordLinks}
-            pillar={pillar}
-            sharingURLs={data.sharingUrls}
-            pageID={data.pageId}
-            isCommentable={data.isCommentable}
-            guardianBaseURL={data.guardianBaseURL}
-        />
-    </InnerContainer>
-);
+}> = ({ pillar, data, config }) => {
+    const tone = data.tags.find(tag => tag.type === 'Tone');
+    const isOpinion = tone ? tone.id === 'tone/comment' : false;
+
+    const topMeta = isOpinion ? (
```

Comment:
This will obviously get more sophisticated once we support other tones too.

Reply:
Agreed, I think we should get this merged ASAP and move onto the next tone. Whatever abstraction we try to define at this point will probably be wrong.

Reply:
Yeah, this is definitely the right approach - abstract after the fact, not before!